Automattic/jetpack
# Update My Jetpack Notice behavior

## Proposed changes:

* This PR adds the possibility of adding a loading text to the actions in a Notice, as well as a `resetNotice` action to `NoticeContext`. While these changes do not affect any of the existing code, it will be used in #36550 and #36592.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Check the code makes sense

If you want to test this functionality:
* Go to the test description of #36550 
* Test how the loading behavior works there
